src/test/java/core/basesyntax/service/WriterServiceTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitInStorage;
+import core.basesyntax.service.impl.CsvWriterServiceImpl;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.Test;
+
+class WriterServiceTest {
+    private static final String RESOURCES_PASS = "src/main/resources/";
+    private WriterService writerService;
+
+    @Test
+    void write_emptyReport_Ok() {
+        writerService = new CsvWriterServiceImpl();
+        List<FruitInStorage> fruits = new ArrayList<>();
+        writerService.write(RESOURCES_PASS + "emptyTest.csv", "");
+        assertEquals("", readFile(RESOURCES_PASS + "emptyTest.csv"));
```

Comment:
Maybe file names better move to static constants and use then without string concatenations